include/faker-cxx/Phone.h

```diff
@@ -25,6 +25,10 @@ class Phone
      * @endcode
      */
     static std::string number(std::optional<std::string> = std::nullopt);
+    static std::string platform();
+    static std::string modelName();
+    static std::string manufacturer();
+
```

Comment:
please add tests for new methods

Reply:
Will I be creating a file (Phone.test.cpp?) with the following code:
#include <faker-cxx/Phone.h>
#include <cassert>
#include <iostream>

void testPlatform() {
    std::string result = faker::Phone::platform();
    assert(!result.empty()); // Check that result is not empty
    // Additional checks can be made if there are specific expected platforms
}

void testModelName() {
    std::string result = faker::Phone::modelName();
    assert(!result.empty()); // Check that result is not empty
    // Additional checks can be made if there are specific expected model names
}

void testManufacturer() {
    std::string result = faker::Phone::manufacturer();
    assert(!result.empty()); // Check that result is not empty
    // Additional checks can be made if there are specific expected manufacturers
}

int main() {
    testPlatform();
    testModelName();
    testManufacturer();
    std::cout << "All Phone tests passed.\n";
    return 0;
}

Or do you want it to be done someway specifically?

Reply:
there is already a test file:
https://github.com/cieslarmichal/faker-cxx/blob/main/src/modules/phone/PhoneTest.cpp

just append new tests in the similar fashion :)

Reply:
do asserts like:
    ASSERT_TRUE(std::ranges::any_of(platforms, [generatedPlatform](const std::string& platform) { return platform == generatedPlatform; }));

Reply:
there is a lot of tests file like that in repository, so you have a references